src/_includes/cwl/hello-world-expressionlib.cwl

```diff
@@ -0,0 +1,25 @@
+cwlVersion: v1.2
+class: CommandLineTool
+requirements:
+  - class: InlineJavascriptRequirement
+    expressionLib:
+      - { $include: functions.js }
```

Comment:
Maybe us a file name that is less generic than `functions.js` to emphasis that this is a user provided value, not a special keyword

Reply:
Without much creativity, I went with `custom-functions.js` :grimacing: